layers/+lang/d/packages.el

```diff
@@ -34,7 +34,10 @@
 
 (defun d/init-flycheck-dmd-dub ()
   (use-package flycheck-dmd-dub :defer t
-    :init (add-hook 'd-mode-hook 'flycheck-dmd-dub-set-include-path)))
+    :init (progn
+            (add-hook 'd-mode-hook 'flycheck-dmd-dub-set-include-path)
+            (add-hook 'd-mode-hook 'flycheck-dmd-dub-set-variables)
```

Comment:
See [this](https://github.com/atilaneves/flycheck-dmd-dub#usage) for the missing setup.